pkg/get/get_test.go

```diff
@@ -3312,38 +3312,56 @@ func Test_DownloadKrew(t *testing.T) {
 		{
 			os:      "darwin",
 			arch:    arch64bit,
-			version: "v0.4.2",
-			url:     `https://github.com/kubernetes-sigs/krew/releases/download/v0.4.2/krew-darwin_amd64.tar.gz`,
+			version: "v0.4.3",
+			url:     `https://github.com/kubernetes-sigs/krew/releases/download/v0.4.3/krew-darwin_amd64.tar.gz`,
 		},
 		{
 			os:      "darwin",
 			arch:    archARM64,
```

Comment:
I think this test may have been wrong in the first place, since the value should be archDarwinARM64 from `uname` on MacOS M1

`Darwin am1.local 21.5.0 Darwin Kernel Version 21.5.0: Tue Apr 26 21:08:29 PDT 2022; root:xnu-8020.121.3~4/RELEASE_ARM64_T8101 arm64`

cc @Jasstkn

Reply:
yes, it should be `archDarwinARM64`.